yarn.lock

```diff
@@ -6479,6 +6479,8 @@ __metadata:
     futil: ^1.76.0
     lodash: ^4.17.21
     minimal-csv-formatter: ^1.0.15
+  peerDependencies:
+    contexture-client: "workspace:^"
```

Comment:
I don't think we should be shipping with `workspace` in our dependencies 🤔

Reply:
Having `workspace` allows you to use the local version when developing instead of pulling a package from NPM. 
We could do it for all the cross-dependencies.
Supposedly changeset should replace workspace with correct version on release
https://github.com/changesets/changesets/issues/432